tests/test_registry.py

```diff
@@ -754,7 +756,7 @@ def test__process_registration_result(self):
         class Mock(Registration):
             def _decrypt_shared_secret(self, encryptor, shared_secret):
                 self._decrypt_shared_secret_called_with = (encryptor, shared_secret)
-                return "cleartext"
+                return "👉 cleartext 👈".encode("utf8")
```

Comment:
We didn't catch this problem earlier because the mock _decrypt_shared_secret returned Unicode, whereas the real method returns a bytestring.